index.bs

```diff
@@ -4457,8 +4457,8 @@ the following table:
    <td class="non-normative">The total size of all the chunks stored in
    [=WritableStreamDefaultController/[[queue]]=] (see [[#queue-with-sizes]])
   <tr>
-   <td><dfn>\[[signal]]</dfn>
-   <td class="non-normative">An {{AbortSignal}} that can be used to abort the pending write or
+   <td><dfn>\[[abortController]]</dfn>
+   <td class="non-normative">An {{AbortController}} that can be used to abort the pending write or
```

Comment:
These slots are sorted alphabetically. So move it up in the table, between `[[abortAlgorithm]]` and `[[closeAlgorithm]]`.

Reply:
Done.